docs/fetching-data.md

```diff
@@ -65,15 +65,39 @@ module.exports = function (api) {
 *..contribute*
 
 ### YAML
-*..contribute*
+Import data from any yaml file to the GraphQL data layer with the [Data store API](/docs/data-store-api/). To use the API you need a `gridsome.server.js` file in the root folder of your Gridsome project.  First, though, you need to install the node packages `fs` and `js-yaml` into your project. Run the following command to install them:
+
+`npm install fs js-yaml --save`
```

Comment:
Should be `--save-dev` since we need it only at build time. By runtime, Gridsome has already converted the YAML to JSON.